umbraco-cloud/getting-started/migrate-existing-site/README.md

```diff
@@ -196,15 +196,16 @@ In this next part, it is time to generate the so-called UDA-files for all your p
 
 For more details about UDA files, read the [UDA Files](../../set-up/power-tools/generating-uda-files.md#what-are-uda-files) article.
 
+Your local Cloud project will need [Umbraco Deploy](https://www.nuget.org/packages/Umbraco.Deploy.OnPrem) installed, you can check your .csproj file to see if this is already installed. If not installed, install this and check your site is working as expected.
```

Comment:
```suggestion

```

Reply:
@lewissmith606 All Umbraco projects come with Deploy out-of-the-box, so there is no need to check whether Deploy is installed 😊

Reply:
Great, I’ll go ahead and remove this line @jonat123